test/verify/check-terminal

```diff
@@ -20,33 +20,34 @@
 import parent
 from testlib import *
 
+def line_sel(i):
+    return '.terminal .xterm-accessibility-tree div:nth-child(%d)' % i
 
 @skipImage("Do not test BaseOS packages", "rhel-8-3-distropkg")
 class TestTerminal(MachineCase):
-
-    @skipBrowser("Firefox needs http to access clipboard", "firefox")
-    @nondestructive
-    def testBasic(self):
-        b = self.browser
-        m = self.machine
-        b.default_user = "admin"
+    def setUp(self):
+        super().setUp()
 
         self.allow_journal_messages(".*external channel failed: terminated")
 
         # Make sure we get what we expect
-        m.write("/tmp/bashrc-extra", """
+        self.machine.write("/tmp/bashrc-extra", """
```

Comment:
Please use `self.write("/home/admin/.bashrc", append=True)` for this. It will correctly deal with cleanup for nondestructive tests, and is simpler to use.

Reply:
Handy. Thanks.